src/eth_dev.rs

```diff
@@ -243,10 +231,24 @@ impl EthDev {
         // SAFETY: `port_id` validity verified
         let errno = unsafe { rte_eth_dev_stop(self.port_id) };
         Error::from_ret(errno)?;
-        debug!("Device {} successfully stopped", self.port_id);
+        log::debug!("Device {} successfully stopped", self.port_id);
         Ok(())
     }
 
+    /// Get a `TxSender`.
+    ///
+    /// # Errors
```

Comment:
There's no error from this function. Maybe it's not an "Error" section in this comment?

Reply:
This comment is changed to explain why a `None` is returned.